### PR TITLE
Itd 5

### LINKED ISSUE #5: Generic repository
Create generic repository and impl it

### PR DESCRIPTION
**Pull request by chizkeks**
Resolves #5 
Generic repository and repositories for model classes created

### CODE REVIEW

File: src/main/java/com/petprojects/todo/repository/CommentRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.petprojects.todo.repository;
+
+import com.petprojects.todo.model.Comment;
+
+public interface CommentRepository extends GenericRepository<Comment> {
+    Comment findByTask_Id(int id);
```
Comment (vkoloskov) [inline src/main/java/com/petprojects/todo/repository/CommentRepository.java:6]: Change to `List<Comment>`

**Review by vkoloskov — COMMENTED** [review]
Change CommentRepository method

File: src/main/java/com/petprojects/todo/repository/CommentRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.petprojects.todo.repository;
+
+import com.petprojects.todo.model.Comment;
+
+import java.util.List;
+
+public interface CommentRepository extends GenericRepository<Comment> {
+    List<Comment> findByTask_Id(int id);
```
Comment (vkoloskov) [inline src/main/java/com/petprojects/todo/repository/CommentRepository.java:8]: findAll?

**Review by vkoloskov — APPROVED** [review]
Approved

**Review by nikitakarnygin — APPROVED** [review]
Ok

**Pull request merged**